promotion/datastore_test.go

```diff
@@ -257,14 +257,18 @@ func (suite *PostgresTestSuite) TestGetAvailablePromotionsForWallet() {
 	suite.Assert().True(promotions[0].Available)
 	suite.Assert().False(promotions[1].Available)
 
-	_, err = pg.CreateClaim(promotion.ID, w.ID, decimal.NewFromFloat(30.0), decimal.NewFromFloat(0))
+	adClaimValue := decimal.NewFromFloat(30.0)
```

Comment:
does it make sense to add another condition to cover a claim value that is too high?

Reply:
what do you mean by too high in this context? higher than `promotion.ApproximateValue`? this isn't the place where we enforce that kind of constaint